custom_components/hafas/config_flow.py

```diff
@@ -32,7 +32,8 @@ class Profile(StrEnum):
 PROFILE_OPTIONS = [
```

Comment:
We should try to factor this out into a config file, so that we can extend it easily. For future PR that is.

Reply:
I opened #8 for this